public/app/features/plugins/components/PluginStateInfo.tsx

```diff
@@ -17,7 +17,7 @@ export const PluginStateInfo = (props: Props) => {
     <Badge
       className={props.className}
       color={display.color}
-      title={display.tooltip}
+      {...(typeof display.tooltip === 'string' ? { title: display.tooltip } : { tooltip: display.tooltip })}
       text={display.text}
       icon={display.icon}
```

Comment:
This is a bit hard to read and understand. Could we do a ternary for each property instead?
title={typeof display.tooltip === 'string' ? display.tooltip : undefined}
tooltip={typeof display.tooltip !== 'string' ? display.tooltip : undefined}